src/Sylius/Behat/Service/CurrentPageResolverInterface.php

```diff
@@ -8,22 +8,20 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
 namespace Sylius\Behat\Service;
 
-use Sylius\Behat\Page\Admin\Crud\CreatePageInterface;
-use Sylius\Behat\Page\Admin\Crud\PageWithFormInterface;
-use Sylius\Behat\Page\Admin\Crud\UpdatePageInterface;
+use Sylius\Behat\Page\PageInterface;
 
 /**
  * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
  */
 interface CurrentPageResolverInterface
 {
     /**
-     * @param CreatePageInterface $createPage
-     * @param UpdatePageInterface $updatePage
+     * @param array $pages
```

Comment:
And more correct if we only need to walk through them all, as it might be `\Traversable` object as well then